src/android/FirebasePluginMessagingService.java

```diff
@@ -101,17 +121,17 @@ private void sendNotification(String id, String title, String messageBody, Map<S
 
             if(android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M)
             {
-				int accentID = getResources().getIdentifier("accent", "color", getPackageName());
-                notificationBuilder.setColor(getResources().getColor(accentID, null));
+                int accentID = getResources().getIdentifier("accent", "color", getPackageName());
+                //notificationBuilder.setColor(getResources().getColor(accentID, null));
```

Comment:
This should probably use ContextCompat:
notificationBuilder.setColor(ContextCompat.getColor(this, accentID);

However, it shouldn't be crashing. If it's crashing, perhaps you don't have an accent color defined in your project. The fix should be checking for null on the getColor() - not comment out the line completely as this breaks everyone that is using this color functionality.